static/js/demo/index.js

```diff
@@ -93,8 +93,8 @@ function startGame(endOfGameCallback) {
 
 
     $("#overcooked").empty();
-    getOvercookedPolicy(players[0], layout_name, 0).then(function(npc_policy_zero) {
-        getOvercookedPolicy(players[1], layout_name, 1).then(function(npc_policy_one) {
+    getOvercookedPolicy(players[0], layout_name, 0, true).then(function(npc_policy_zero) {
+        getOvercookedPolicy(players[1], layout_name, 1, true).then(function(npc_policy_one) {
```

Comment:
Is this setting it to be `argmax`ing? I think we should have sampling by default

Reply:
This change makes it sample, but it's true that the default is still argmax; I can change that. I think my earlier prior was to have it by-default do what it was doing before

Reply:
Oh I think I had previously changed it to be default to sample, but maybe the change was overwritten (?). Yeah I think if we could change it to default to sample it would be better. Thanks Cody!